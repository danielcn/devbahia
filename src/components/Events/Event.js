import React, {Component} from 'react'

class Event extends Component {
  constructor(props) {
    super(props);

    this.state = {events: []};
  }

  componentDidMount() {
    this.eventList();
  }

  eventList() {
    $.getJSON('https://localhost:88888')
      .then(({ results }) => this.setState({ person: results }));
  }

  render() {
    const events = this.state.events.map((item, i) => (
      <div>
        <h1>{ item.name.first }</h1>
        <span>{ item.cell }, { item.email }</span>
      </div>
    ));

    return (
      <div id="layout-content" className="layout-content-wrapper">
        <div className="panel-list">{ events }</div>
      </div>
    );
  }
}

export default Event;
