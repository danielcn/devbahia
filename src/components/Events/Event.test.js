import React from 'react';
import { shallow } from 'enzyme';
import Event from './Event';

describe('Event Snapshot', () => {
  test('renders', () => {
    const event = shallow(<Event />);
    expect(event).toMatchSnapshot();
  });
});
