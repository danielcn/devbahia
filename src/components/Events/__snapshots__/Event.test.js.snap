// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event Snapshot renders 1`] = `ShallowWrapper {}`;
